ores/wsgi/routes/v3/precache.py

```diff
@@ -14,15 +14,15 @@ def configure(config, bp, scoring_system):
 
     precache_map = util.build_precache_map(config)
 
-    @bp.route("/v3/precache/", methods=["GET"])
+    @bp.route("/v3/precache/", methods=["POST"])
     @preprocessors.nocache
     @preprocessors.minifiable
     def precache_v3():
-        if 'event' not in request.args:
+        if 'event' not in request.form:
```

Comment:
I don't think we need this at all. The body can just be the event itself in JSON format.